docs/README.md

```diff
@@ -78,6 +78,14 @@ docker run --rm `
   jekyll/jekyll `
   jekyll serve
 ```
+> Note: You may seen a warning saying `Auto-regeneration may not work on some Windows versions.` Symptoms of this warning are the inability to automatically see any local website changes in your browser. This makes website changes difficult as you must restart Jekyll after every local change to view the results. To work around this issue, try the `--force-polling` switch to the `jekyll serve` command.
```

Comment:
nit: can you please line-wrap such long comments? there's shortcuts/tools/plugins for every major IDE, e.g. Option+Q in vscode.

Reply:
Opt+Q doesn't seem to work out of the box, but there's an extension called "Rewrap" that uses Option+Q and seems to do what you're asking for.

Reply:
Ah, wasn't sure whether it's builtin or extension. Cool!